package.json

```diff
@@ -1832,7 +1832,7 @@
             "panel": [
                 {
                     "id": "jupyter-variables",
-                    "title": "Jupyter",
+                    "title": "",
```

Comment:
This seems weird to me. If I drag the panel then it won't have a name anymore.

Reply:
We raced 😂

Reply:
![image](https://user-images.githubusercontent.com/20347396/109737650-3c6e7d80-7b7b-11eb-8258-2438f79910df.png)

It still has the full name, and for @joyceerhl, the tooltip is still there.

Reply:
So the ":" in the name was actually just an effect of having both a view container name and a view name. I didn't explicitly add the ":". So if we go with this approach of not giving the view a name, then we could probably just call it "Jupyter Variables" the ":" isn't needed.